### DISCUSSION TITLE
Is there a concern for the size of the "yourls_log" table? Mine is 25GB with over 266 million entries.

### DISCUSSION THREAD

**Discussion by joehmphack** [YOURLS/YOURLS#3739] [General]
Should I be concerned with a very large "yourls_log" table? Mine is currently 25.5 GiB with over 266,000,000 entries. I have over 500,000 tracked links.

Are there any options/plugins for truncating the logs without losing too many stats details?

**Comment by ehthxC7x9D** [comment]
I've typically just truncated the entire `YOURLS_DB_TABLE_LOG` table when I was running a more active service.  Your URL table will still contain the aggregated click counts for each link you've created, but will no longer have the logs of who clicked what, when, where, etc.
